### ISSUE TITLE
Menubar obscuring content

### ISSUE THREAD

**Issue by thortek** [UVU-DigitalMedia/DGM-Competency-Browser#36]
There is content stuffed underneath the menubar.  Need some CSS probably to bring it down from being hidden there.

**Comment by clintlosee** [comment]
I ran into this same issue and once I ran 'grunt sass' command it fixed the issue.

**Comment by thortek** [comment]
Yup, that fixed it.  We should probably add that step into the ReadMe file.
